docs/UserGuide.md

```diff
@@ -315,14 +319,15 @@ Format: `clear`
 
 Adds a meeting to your meeting list.
 
-Format: `add t/TITLE [d/DESCRIPTION] from/FROM to/TO [c/CONTACTS] [l/LOCATION]`
+Format: `add t/TITLE from/FROM to/TO [d/DESCRIPTION] [c/CONTACTS] [l/LOCATION]`
 * `TITLE` is the main heading of the meeting.
-* `DESCRIPTION` contains additional details about the meeting, e.g. agenda. 
 * `FROM` is the start date time of the meeting in DD-MM-YYYY HH:mm format.
 * `TO` is the end date time of the meeting in DD-MM-YYYY HH:mm format.
+* `DESCRIPTION` contains additional details about the meeting, e.g. agenda.
 * `CONTACTS` represents the contacts involved in the meeting. 
 * `CONTACTS` is a comma-separated string of names.
 * `LOCATION` is the location of the meeting.
+* As you might want to take note of the meetings that your team has, meetings can overlap.
```

Comment:
I got a different impression about the reasoning for this overlapping issue. 
I thought that the reason should be "Meeting timings may overlap as you may wish to send a representative for your clashing meetings". Something along these lines.

May want to consider putting this as a `Note` (i.e. 
![image](https://user-images.githubusercontent.com/58500317/97977725-99456b80-1e07-11eb-936e-1e5283d9ea57.png))